substrate/frame/staking/reward-curve/src/lib.rs

```diff
@@ -79,14 +79,14 @@ pub fn build(input: TokenStream) -> TokenStream {
 	let declaration = generate_piecewise_linear(points);
 	let test_module = generate_test_module(&input);
 
-	let imports = match crate_name("sp-runtime") {
```

Comment:
This doesn't seem to work (see CI errors)

Reply:
ok, I realized that this is just a big rabbit hole if I do it this way since this would affect all other pallets using this proc macro. 

I instead made it backwards compatible. It will work for crates that are either using the `sp-runtime` or `polkadot-sdk-frame` dependencies.